internal/service/cloudbackupschedule/resource_cloud_backup_schedule.go

```diff
@@ -492,9 +492,9 @@ func cloudBackupScheduleCreateOrUpdate(ctx context.Context, connV2 *admin.APICli
 	}
 
 	req := &admin.DiskBackupSnapshotSchedule{}
-
-	if v, ok := d.GetOk("copy_settings"); ok && len(v.([]any)) > 0 {
-		req.CopySettings = expandCopySettings(v.([]any))
+	copySettings := d.Get("copy_settings")
+	if len(copySettings.([]any)) > 0 || d.HasChange("copy_settings") {
```

Comment:
you may want to use: conversion.HasElementsSliceOrMap

Reply:
thanks updated in f9d2e6964d64617b60dc4145853fe9c329533b68